README.md

```diff
@@ -1,3 +1,41 @@
-# Deprecated
+# [helm-chart](https://github.com/jupyterhub/helm-chart)
 
-This repository has been merged into the [Zero to JupyterHub](https://github.com/jupyterhub/zero-to-jupyterhub-k8s) repository.
+[![Join the chat at https://gitter.im/jupyterhub/binder](https://badges.gitter.im/jupyterhub/binder.svg)](https://gitter.im/jupyterhub/binder?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+This repository contains the Helm charts for BinderHub and JupyterHub. The
```

Comment:
Perhaps say 'Helm chart tarball repository'? Just to make it clear that actual development doesn't happen here, just the built tarballs

Reply:
Good point!